### PR TITLE
Rkravitz gaussian initial impulse

### PR DESCRIPTION
**Pull request by rkrav**
The initial infection source was modeled as an impulse.  Now it's modeled as a gaussian curve, with the same total area as before.

### CODE REVIEW

**Comment by wbunting** [comment]
👍  LGTM

File: model/model.wl [hunk 1]
```diff
@@ -103,8 +114,11 @@ containmentThresholdCases_,
 icuCapacity_,
 distancing_,
 hospitalCapacity_:1000000, (*defaulted since we dont evaluate this on a country basis yet *)
-tMin_
+tMin_,
+est_
```
Comment (rkrav) [inline model/model.wl]: oops this is wrong

Comment (wbunting) [reply model/model.wl]: oh!

File: model/model.wl [hunk 2]
```diff
@@ -81,6 +81,18 @@ pS0 = 1-(pC0 + pH0);
...
+getEst[importLength_, initialInfectionImpulse_, importtime_] := 
+	Module[{totalInfection, sig, mu, areaUnderCurve},
+		totalInfection = importLength*Exp[-initialInfectionImpulse];
+		(* the normal should hit the boundaries at +/- 3 standard deviations *)
+		sig = importLength / 6;
+		mu = importtime + .5*importLength;
+		(* area under gaussian with standard_deviation = sigma *)
+		areaUnderCurve = sig*Sqrt[2*Pi];
+		With[{mu0 = mu, sig0 = sig, ti = totalInfection, auc = areaUnderCurve},
+			Function[t, Piecewise[{{ti/auc * PDF[NormalDistribution[mu0, sig0], t],importtime <= t <= importtime + importLength}} ]]]]
```
Comment (rkrav) [inline model/model.wl]: I added this piecewise term to try and avoid an underflow from evaluating the gaussian far away from its center point.  This doesn't seem to be working though.

**Comment by wbunting** [comment]
Thanks @rkrav ! turns out this was not the major speed bottleneck... going to close for now to keep the model in-line with what Harvard has

**Pull request closed without merging**